drivers/led/pca9633.c

```diff
@@ -43,15 +43,20 @@ LOG_MODULE_REGISTER(pca9633);
 
 #define PCA9633_MASK            0x03
 
-struct pca9633_data {
+struct pca9633_cfg {
 	const struct device *i2c;
```

Comment:
While at it it would be worth converting this driver to use i2c_dt_spec.

Reply:
Sure, do you have an example i can work from?

Reply:
Here you go https://github.com/zephyrproject-rtos/zephyr/pull/39901

Reply:
Updated the driver to use i2c_dt_spec